app/build.gradle

```diff
@@ -92,11 +92,14 @@ dependencies {
 
     implementation 'com.crashlytics.sdk.android:crashlytics:2.10.1'
 
+    implementation 'com.google.android.gms:play-services-location:17.0.0'
+
     testImplementation "org.spekframework.spek2:spek-dsl-jvm:$spek2Version"
     testImplementation "org.spekframework.spek2:spek-runner-junit5:$spek2Version"
     testImplementation "org.assertj:assertj-core:$assertJVersion"
     testImplementation "io.mockk:mockk:$mockkVersion"
     testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:$coroutinesVersion"
 }
 
-apply plugin: 'com.google.gms.google-services'
\ No newline at end of file
+apply plugin: 'com.google.gms.google-services'
+apply plugin: 'kotlin-android-extensions'
```

Comment:
I think you do not use this.
If you use it could you move it with all other plugin